manifests/proxydhcp.pp

```diff
@@ -49,6 +49,25 @@
     failover       => $failover,
   }
 
+
+  if $foreman_proxy::dhcp_manage_acls and $::osfamily == 'RedHat' {
```

Comment:
Maybe move the osfamily check to params.pp?

Reply:
And set `dhcp_manage_acls` to false or true based on that? Will that require doing a `dhcp_manage_acls_real` variable that checks both osfamily and whatever the user specified?

Reply:
Yes, since you already set it to true in params.pp you can remove the check here. This can become just ```if $::foreman_proxy::dhcp_manage_acls```.

Reply:
So the assumption then is if the user sets this to `true` and are on say, debian, that they just have to deal with the consequences themselves of setting it to true on a system that doesn't support this?

Reply:
Yes, on Debian you can also use ACLs and maybe we should allow puppet-dhcp to tighten the permissions to 0750 so we're more secure by default (introduce ```dhcp_dir_mode```, default to 0755 like we do now). Then it's a feature they would want.